ogr/services/gitlab/issue.py

```diff
@@ -112,6 +120,7 @@ def get(project: "ogr_gitlab.GitlabProject", id: int) -> "Issue":
 
     @staticmethod
     def get_list(
+        self,
```

Comment:
Probably some rebase problem? Probably should not be in a static method.

Reply:
Yup fixed this.